arangod/Transaction/Manager.cpp

```diff
@@ -123,14 +119,11 @@ void Manager::unregisterTransaction(TransactionId transactionId,
                                     bool isReadOnlyTransaction,
                                     bool isFollowerTransaction) {
   // always perform an unlock when we leave this function
-  auto guard = scopeGuard([this, transactionId, &isReadOnlyTransaction,
+  auto guard = scopeGuard([transactionId, &isReadOnlyTransaction,
```

Comment:
This guard seems unnecessary, since it does not do anything except TRACE logging something which is no longer relevant. Remove?